features/globalSearch/GlobalSearchDesktop.tsx

```diff
@@ -42,7 +42,8 @@ const GlobalSearchDesktop = () => {
     >
       <IconButton
         color="inherit"
-        aria-label={t("search.toggleSearch")}
+        // i18n leads to unknown server/client mismatch error
```

Comment:
I've been seeing this one, does this change solve it? I'm a bit stumped by this error, unfortunately!